components/brave_vpn/resources/panel/state/actions.ts

```diff
@@ -32,6 +32,7 @@ export const connectionFailed = createAction('connectionFailed')
 export const retryConnect = createAction('retryConnect')
 export const initialize = createAction('initialize')
 export const purchaseConfirmed = createAction('purchaseConfirmed')
+export const initUISell = createAction('initUISell')
 
 export const initUIMain = createAction<InitUIMainPayload>('initUIMain')
```

Comment:
Perhaps we should explicitly let the actions describe the event that's happened - in this case be clearer that we have initialized and retrieved all the data required for the "main" state. At the moment it seems like a command to initialize and go and get the data.

```
1. initialize // gets is purchased or not
2. purchaseConfirmed(isPurchased: true | false) // pass isPurchased only in future if you need data for sell panel
3. mainDataReady // or SHOW MAIN VIEW
3. sellDataReady // or SHOW SELL VIEW only in future if need data for sell panel
```

Reply:
Or just change initUISell to include the word `initialized`